mail_drop_target/__manifest__.py

```diff
@@ -8,9 +8,13 @@
     "category": "Discuss",
     "website": "https://github.com/OCA/social",
     "summary": "Attach emails to Odoo by dragging them from your desktop",
-    "depends": ["mail"],
+    "depends": ["mail", "mass_mailing"],
```

Comment:
We don't want to force those using `mail_drop_target` to install `mass_mailing`.

I the dependency is needed because what you fix is only happening when `mass_mailing` is installed, then you need to submit a new "glue" module.

Reply:
@grindtildeath  thanks for review, replaced with new module